data/presets/presets/leisure/marina.json

```diff
@@ -3,24 +3,28 @@
     "fields": [
         "name",
         "operator",
-        "capacity",
+        "phone",
+        "email",
+        "website",
         "fee",
         "payment_multi_fee",
         "charge_fee",
         "sanitary_dump_station",
-        "power_supply"
+        "internet_access",
+        "internet_access/fee",
+        "maxlength",
+        "maxwidth",
+        "maxdraft",
+        "maxairdraft",
+        "power_supply",
+        "vhf_channel"
```

Comment:
I don't think all these fields exist yet. You'll have to add them in files under `data/presets/fields`.